charts/alfresco-share/Chart.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v2
+name: alfresco-share
+description: Alfresco Share Helm chart for Kubernetes
+type: application
+version: 0.1.0-alpha.0
+appVersion: 7.4.0
+dependencies:
+  - repository: https://alfresco.github.io/alfresco-helm-charts
+    version: 2.1.0-alpha.0
```

Comment:
remember after test to release also common (avoid having a stable chart using an unstable dep)

Reply:
that chart is also alpha. Will do it only only stable release

Reply:
I would not tie that to the sprint, just have the version to be a reflection of the current state we think the chart is  (after testing on our integration pipeline we release a stable after an alpha)